Project.toml

```diff
@@ -45,7 +45,7 @@ ClimaComms = "0.6.4"
 ClimaCore = "0.14.19"
 ClimaDiagnostics = "0.2.4"
 ClimaParams = "0.10.17"
-ClimaTimeSteppers = "0.7.38"
+ClimaTimeSteppers = "0.8"
```

Comment:
Maybe don't change this in this PR for clarity, or rebase after https://github.com/CliMA/ClimaAtmos.jl/pull/3542 is in.

Reply:
Yes, I'm waiting for it